StarVMC/Geometry/FstmGeo/FstmGeoS.xml

```diff
@@ -0,0 +1,1089 @@
+            <var name="version"   type="float" />
+            <var name="z"         type="float" />
+            <var name="rmin"      type="float" />
+            <var name="rmax"      type="float" />
+            <var name="phi1"      type="float" />
+            <var name="phi2"      type="float" />
+            <var name="dz"        type="float" />
+        </Structure>
+
+        <Structure name="coolingtubes" comment="cooling tube (straight part)">
+            <var name="version"   type="float" />
+            <var name="rmin"      type="float" />
+            <var name="rmax"      type="float" />
+            <var name="phi1"      type="float" />
+            <var name="phi2"      type="float" />
+            <var name="dz(7)"     type="float" />
+            <var name="x(7)"      type="float" />
+            <var name="y(7)"      type="float" />
+            <var name="z(7)"      type="float" />
+            <var name="rotz(7)"   type="float" />
+            <var name="rotx(7)"   type="float" />
+        </Structure>
+
+        <Structure name="heatsink" comment="Heat sinks">
+            <var name="version"   type="float" />
+            <var name="dx(8)"    type="float" />
+            <var name="dy(8)"    type="float" />
+            <var name="dz(8)"    type="float" />
+            <var name="x(8)"     type="float" />
+            <var name="y(8)"     type="float" />
+            <var name="z(8)"     type="float" />
+            <var name="rotz(8)"  type="float" />
+        </Structure>
+
+        <Structure name="apvchip" comment="APV chips">
+            <var name="version"   type="float" />
+            <var name="dx(8)"     type="float" />
+            <var name="dy(8)"     type="float" />
+            <var name="dz"        type="float" />
+            <var name="x(8)"      type="float" />
+            <var name="y(8)"      type="float" />
+            <var name="z(8)"      type="float" />
+            <var name="rotz(8)"   type="float" />
+        </Structure>
+
+        <Structure name="tboard" comment="T-board">
+            <var name="version"   type="float" />
+            <var name="dx"        type="float" />
+            <var name="dy"        type="float" />
+            <var name="dz"        type="float" />
+            <var name="x"         type="float" />
+            <var name="y"         type="float" />
+            <var name="z"         type="float" />
+        </Structure>
+
+        <Structure name="connectors" comment="T-board connectors">
+            <var name="version"   type="float" />
+            <var name="dx"        type="float" />
+            <var name="dy"        type="float" />
+            <var name="dz(2)"     type="float" />
+            <var name="x(2)"      type="float" />
+            <var name="y(2)"      type="float" />
+            <var name="z(2)"      type="float" />
+        </Structure>
+        
+        <Structure name="outerhybrida" comment="outer hybrid a">
```

Comment:
Outer and inner hybrids A vs B is where starsim would have a problem.  If you think you might want to run a simulation based on this geometry, you should consider renaming these structures.